### DISCUSSION TITLE
Discussion cards on Android, wasted space

### DISCUSSION THREAD

**Discussion by ciruz** [community/community#9084] [Mobile] [answered]
The discussion cards on Android are not as informative and useful as on iOS.

Android:

![Screenshot_20211219-112156~2.jpg](https://user-images.githubusercontent.com/987947/146671774-93a7f3bb-25a7-46d0-b33f-a1feece6b5a6.jpg)

iOS:
![IMG_0035~2.PNG](https://user-images.githubusercontent.com/987947/146672117-430acf0b-421e-478d-a0a3-54fa130f4163.PNG)

**Comment by mxie** [comment, marked as answer]
Thanks for sharing your feedback @ciruz! We've relayed this back to our team to check out.
